credentials/README.md

```diff
@@ -8,14 +8,81 @@ import "github.com/dfns/dfns-sdk-go/credentials"
 
 ## Index
 
+- [func WithAWSKMSClient\(client AWSKMSClient\) func\(\*AWSKMSSigner\)](<#WithAWSKMSClient>)
+- [type AWSKMSClient](<#AWSKMSClient>)
+- [type AWSKMSSigner](<#AWSKMSSigner>)
+  - [func NewAWSKMSSigner\(ctx context.Context, cfgSigner \*AWSKMSSignerConfig, options ...func\(\*AWSKMSSigner\)\) \(\*AWSKMSSigner, error\)](<#NewAWSKMSSigner>)
+  - [func \(akss \*AWSKMSSigner\) Sign\(userActionChallenge \*credentials.UserActionChallenge\) \(\*credentials.KeyAssertion, error\)](<#AWSKMSSigner.Sign>)
+- [type AWSKMSSignerConfig](<#AWSKMSSignerConfig>)
 - [type AsymmetricKeySigner](<#AsymmetricKeySigner>)
   - [func NewAsymmetricKeySigner\(config \*AsymmetricKeySignerConfig\) \*AsymmetricKeySigner](<#NewAsymmetricKeySigner>)
   - [func \(signer \*AsymmetricKeySigner\) Sign\(userActionChallenge \*credentials.UserActionChallenge\) \(\*credentials.KeyAssertion, error\)](<#AsymmetricKeySigner.Sign>)
 - [type AsymmetricKeySignerConfig](<#AsymmetricKeySignerConfig>)
 
 
+<a name="WithAWSKMSClient"></a>
+## func [WithAWSKMSClient](<https://github.com/vincentserpoul/dfns-sdk-go/blob/main/credentials/aws_kms_signer.go#L34>)
```

Comment:
nit: https://github.com/dnfs/dfns-sdk-go not https://github.com/vincentserpoul/dfns-sdk-go

Reply:
(other occurrences bellow)